src/main/resources/react4xp/_entries/KpiCalculator.jsx

```diff
@@ -151,7 +151,8 @@ function KpiCalculator(props) {
 
   function isEndMonthValid(value) {
     const endMonthValue = value || endMonth.value
-    const endMonthValid = !((endYear.value === validMaxYear) && (endMonthValue > validMaxMonth))
+    const maxYearAverage = Number(validMaxMonth) === 12 ? validMaxYear : Number(validMaxYear) - 1
```

Comment:
er "average" et godt navn her? Det ser ut som det du gjør er å korrigere for siste måned i år, stemmer ikke det?

Reply:
endMonthCorrected kanskje? 
Det korrigeres jo for siste måned i året?

Reply:
Denne variablen viser bare hvilket år man kan beregne gjennomsnitt av, så siste år man kan gjøre det er 2021 siden vi har tall for desember for dette året.